style.css

```diff
@@ -131,4 +132,291 @@ img {
+    text-align: center;
+}
+
+.grand {
+    color: var(--color-accent-red);
+}
+
+.players {
+    color: var(--color-accent-yellow);
+}
+
+.about-main-part{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 20px;
+    height: 180px;
+}
+
+.about-main-part-text{
+    max-width: 460px;
+    margin: 0;
+}
+
+.img-live{
+    width: 320px;
+    height: 180px;
+}
+
+@media(max-width:750px){
+    .about{
+        padding: 40px 20px;
+    }
+
+    .about-main-copy{
+        font-size: 24px;
+    }
+
+    .about-main-part{
+        gap: 0;
+    }
+
+    .about-main-part-text{
+        font-family: var(--font-family-about-mobile);
+        max-width: 750px;
+    }
+
+    .about-main-part-img-wrapper{
+        display: none;
+        width: 0px;
+    }
+}
+
+/*Courses*/
+.courses {
+    margin: 0;
+    background-color: var(--color-background-naby);
+    padding: 80px 20px;
+    color: var(--color-white);
+    font-family: var(--font-family-body);
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+}
+
+.courses-container{
+    max-width: 800px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+}
+
+.courses-title{
+    margin: 0;
+    font-size: 32px;
+    font-family: var(--font-family-heading);
+    font-weight: 400;
+    letter-spacing: 0.02em;
+    text-align: center;
+}
+
+/*Cards*/
+.courses-cards-container{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    gap: 20px;
+}
+
+.courses-cards{
+    background-color: var(--color-accent-yellow);
+    box-shadow: 4px 4px 0px var(--color-accent-red);
+    height: 150px;
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    text-decoration: none;
+    transition-property: "background-color" "box-shadow";
+    transition-duration: 300ms;
+    transition-timing-function: linear;
+}
+
+.courses-cards:hover{
+    background-color: var(--color-course-hover-yellow);
+    box-shadow: 12px 12px 0px var(--color-button-hover-red);
+}
+
+.courses-cards-img-wrapper{
+    background-image: url("./img/live.png");
+    max-width: 250px;
+    min-width: 250px;
+    height: 100%;
+    background-size: cover;
+    position: relative;
+}
+
+.courses-cards-img-overlay{
+    width: 100%;
+    height: 100%;
+    background-color: var(--color-course-overlay);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.courses-cards-title{
+    line-height: 48px;
+    margin: 0;
+    font-size: 32px;
+    font-family: var(--font-family-heading);
+    font-weight: 400;
+    letter-spacing: 0.05em;
+    color: var(--color-white);
+}
+
+.courses-cards-text-container{
+    margin: 0;
+    padding: 10px 20px;
+    font-size: 16px;
+    display: block;
+    flex-direction: column;
+    gap: 10px;
+    height: 100%;
+}
+
+.courses-cards-text{
+    margin: 0;
+    color: var(--color-black);
+    height: 108px;
+}
```

Comment:
```suggestion
.courses-cards-text{
    margin: 0;
    color: var(--color-black);
}
```

textの高さを固定値でもってしまうと、html側で中の文字が変化した際にレイアウト崩れが生じる可能性があります。
このような内容物が変化する要素については、heightは指定せずに内容物に合わせて伸縮してもらうのが一般的かなと思います。

Reply:
そうすると「もっと見る」も上に上がってしまうのですがどうすればいいでしょうか？

Reply:
![image](https://github.com/JunPlayGround/Frontend-Basic-SIte/assets/83479175/75f51501-851f-42f1-a40a-83b61f267e0e)
親要素に `justify-content: space-between;` を設定するのがいいかと思います。
黒い文字が上端に、もっと見るが下端に寄ります。
参考：https://developer.mozilla.org/ja/docs/Web/CSS/justify-content

Reply:
わかりました
ようやく解決しました